landing-page/src/WebView.tsx

```
@@ -0,0 +1,759 @@
+        <Stack width={fixWidth} flexDirection="row" justifyContent="space-between">
+          <Stack justifyContent="center">
+            <img src={madanilogo} alt="madani" width="240px" height="92px" />
+          </Stack>
+          <Stack gap="24px" width="637px">
+            <Typography fontSize="48px" fontWeight="600" color="#00254D">{madaniInfo.title}</Typography>
+            <Stack gap="16px">
+              <Typography fontSize="20px" fontWeight="600" color="#11181C">{madaniInfo.subtitle}</Typography>
+              <Stack gap="4px">
+                {madaniInfo.description.map(point => (
+                  <Stack flexDirection="row" gap="4px" alignItems="center">
+                    <CheckCircle sx={{color: "#ED5F00"}}/>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)